backend/src/main/java/org/sefglobal/core/academix/model/CategoryTranslation.java

```diff
@@ -27,10 +30,10 @@
     private Category category;
 
     @Id
-    @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "language_id",
-                referencedColumnName = "id",
-                nullable = false)
+    @Enumerated(EnumType.STRING)
+    @Column(name = "language",
```

Comment:
No need to specify the name if it should be same as the variable name

Reply:
Ah right. Updated